protocol/messenger_messages.go

```diff
@@ -0,0 +1,108 @@
+package protocol
```

Comment:
Will there be more functions moved to this file? Otherwise `messenger_messages.go` name sounds counterintuitive. Since there are only a couple of functions related particularly to editing

Reply:
there will be, yes, I haven't moved all of them to avoid too many changes